src/README.md

```diff
@@ -8,7 +8,7 @@ as registry.access.redhat.com/rhscl/python-{{ spec.short_ver }}-rhel7
 and the RHEL8 image as [registry.access.redhat.com/rhel8/python-{{ spec.short_ver }}](https://access.redhat.com/containers/#/registry.access.redhat.com/rhel8/python-{{ spec.short_ver }}).
 The CentOS7 image is then available on [Docker Hub](https://hub.docker.com/r/centos/python-{{ spec.short_ver }}-centos7/)
 as centos/python-{{ spec.short_ver }}-centos7.
-The resulting image can be run using [Docker](http://docker.io).
+The resulting image can be run using [podman](https://github.com/containers/libpod).
```

Comment:
Could we keep also Docker here? I don't know how many people actually know podman so it would be nice to keep both technologies here and use it to tell that podman is something similar to Docker for those who don't know it yet.

Reply:
Agreed, we need at least a transition period.

Reply:
We only left the mention to the actual docker application because of there not being a podman at all in rhel7. Now that podman is available in rhel 7.6 I personally would like to remove the mention of docker completely. Imo just having the link to the podman reppository, which has pretty good documentation (including [the comparison of docker to podman](https://github.com/containers/libpod/blob/master/transfer.md)) should be enough.

@hhorak WDYT about keeping a note somewhere in the README that podman usage is equivalent to docker?

Reply:
Checked offline with @hhorak and he is in favour of leaving a note in the README as well. So will add one to the beginning of the file.

Reply:
So you please we add a `or podman` on that line? So that if people skim just the initial paragraph, they know they can use both.